data/pathfinder/dreamscarred_press/psionics_expanded/psionics_expanded_races.lst

```diff
@@ -6,8 +6,8 @@ SOURCELONG:Psionics Expanded	SOURCESHORT:PsiEx	SOURCEWEB:http://dreamscarredpres
 # NEEDS CODE WORK: The Swarm template didn't give the Swarm Traits and
 # NEEDS CODE WORK:   Distraction abilities
 # Race Name		Starting Feats	Size		Move			Face/Space	Reach		Vision			Combat bonus				Stat bonus								Modify VAR			Weapon prof. bonus				Ability																				Nb Legs	Nb Hands	Natural Attacks									Monster Class Name and Starting Level	Main Race Type		Template		Challenge Rating	Source Page		Apply Kit
-Astral Swarm				SIZE:D	MOVE:Walk,30	FACE:10	REACH:0					BONUS:COMBAT|AC|2|TYPE=NaturalArmor	BONUS:STAT|STR|-9	BONUS:STAT|DEX|8				BONUS:VAR|Mindless|1	BONUS:WEAPONPROF=Swarm|DAMAGE|-1*(STR)	ABILITY:Special Ability|AUTOMATIC|Distraction ~ Swarm|Ectoplasmic Poison ~ Ectoplasmic Swarm|No Combat Maneuvers|Swarm Traits					NATURALATTACKS:Swarm,Weapon.Natural.Melee.Swarm.Poison,*1,4d6	MONSTERCLASS:Construct:14			RACETYPE:Construct	TEMPLATE:Swarm	CR:7			SOURCEPAGE:p.66
-Astral Warrior	STARTFEATS:0	SIZE:M	MOVE:Walk,20	FACE:5	REACH:5	VISION:Darkvision (60')	BONUS:COMBAT|AC|2|TYPE=NaturalArmor	BONUS:STAT|STR|6	BONUS:STAT|DEX|2	BONUS:STAT|CON|2																																	LEGS:2	HANDS:2												MONSTERCLASS:Astral Warrior:1			RACETYPE:Construct							SOURCEPAGE:p.66	KIT:1|Astral Warrior
+Astral Swarm				SIZE:D	MOVE:Walk,30			REACH:0					BONUS:COMBAT|AC|2|TYPE=NaturalArmor	BONUS:STAT|STR|-9	BONUS:STAT|DEX|8				BONUS:VAR|Mindless|1	BONUS:WEAPONPROF=Swarm|DAMAGE|-1*(STR)	ABILITY:Special Ability|AUTOMATIC|Distraction ~ Swarm|Ectoplasmic Poison ~ Ectoplasmic Swarm|No Combat Maneuvers|Swarm Traits					NATURALATTACKS:Swarm,Weapon.Natural.Melee.Swarm.Poison,*1,4d6	MONSTERCLASS:Construct:14			RACETYPE:Construct	TEMPLATE:Swarm	CR:7			SOURCEPAGE:p.66
```

Comment:
SIZE:D will trigger FACE: equivalent to 1,1.... so I don't think this has the desired effect (will all of the swarms through Pathfinder have this same issue?)

Reply:
Any Race with RACESUBTYPE:Swarm or TEMPLATE:Swarm will have the Swarm FACE applied.